docs/app/page.tsx

```
@@ -0,0 +1,279 @@
+                                    className={"text-decoration-none"}
+                                    href={"https://github.com/y-scope/clp/releases"}
+                                    rel={"noreferrer noopener"}
+                                    target={"_blank"}
+                                >
+                                    <svg
+                                        fill={"currentColor"}
+                                        height={"25"}
+                                        viewBox={"0 0 24 24"}
+                                        width={"25"}
+                                        xmlns={"http://www.w3.org/2000/svg"}
+                                    >
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)